gematria/datasets/find_accessed_addrs_exegesis.cc

```diff
@@ -211,19 +221,8 @@ Expected<ExecutionAnnotations> ExegesisAnnotator::findAccessedAddrs(
     MemAnnotations.add_accessed_blocks(Mapping.Address);
   }
 
-  std::vector<unsigned> UsedRegisters = gematria::getUsedRegisters(
-      *DisInstructions, State.getRegInfo(), State.getInstrInfo());
-
   MemAnnotations.mutable_initial_registers()->Reserve(UsedRegisters.size());
```

Comment:
This line should move along with the rest of the code (it's pre-allocating memory for all the `add_initial_regisers()` calls that were in the for loop below.